pkg/compact/downsample_test.go

```diff
@@ -0,0 +1,114 @@
+package compact
+
+import (
+	"testing"
+
+	"github.com/improbable-eng/thanos/pkg/testutil"
+	"github.com/prometheus/tsdb/labels"
+)
+
+func TestDownsampleAggregator(t *testing.T) {
+	var cases = []struct {
+		lset labels.Labels
+		data []sample
+		exp  []series
+	}{
+		{
+			lset: labels.FromStrings("__name__", "a"),
+			data: []sample{
+				{20, 1}, {40, 2}, {60, 3}, {80, 1}, {100, 2}, {120, 5}, {180, 10}, {250, 1},
+			},
+			exp: []series{
+				{
+					lset: labels.FromStrings("__name__", "a$sum"),
```

Comment:
why `$` at the end?

Reply:
That's what gets generated as a downsampling series.

Reply:
mhm. sure, but any reason, why  not `@` as was in initial idea? (:

Reply:
Mh, it's just what ended up being used when discussing with Brian. I suppose `@` has some kind of different association of relationship, like `metric@5m` or `metric@<job>`.

Reply:
ok just curious :+1: